roles/update_containers/tasks/main.yml

```diff
@@ -14,21 +14,23 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-- name: Destination directory exists
-  ansible.builtin.file:
-    state: directory
-    recurse: true
-    path: "{{ cifmw_update_containers_dest_path | dirname }}"
-
-- name: Generate update containers yaml file
-  ansible.builtin.template:
-    src: "update_containers.j2"
-    dest: "{{ cifmw_update_containers_dest_path }}"
-
```

Comment:
@bshewale Thank you for the patch. The main objective of these two tasks is to allow to generate the container updates cr file only.(It will be useful for anyone using this role standalone.)

Apply only when users want to apply that.